ros/src/util/packages/autoware_launcher/documents/dev/devmode.md

```diff
@@ -0,0 +1,10 @@
+# Developer Mode
+
+Developer Mode can be switched by (Window Menu -> View -> Develop Mode).<br>
+![devmode](./devmode.png)
+
+NOTICE
+* On treeview widget, R key is run, T key is terminate.
+* On summary panel, I key initialize the parameters to default.
+* Use "save as" instead of "save".
```

Comment:
what do you mean with this?

Reply:
Updated as following.
`Currently Save Profile from the File menu does not work. Use Save Profile As instead.`